Networking/Networking/Model/Copiable/Models+Copiable.generated.swift

```diff
@@ -67,7 +67,7 @@ extension Order {
         let customerNote = customerNote ?? self.customerNote
         let dateCreated = dateCreated ?? self.dateCreated
         let dateModified = dateModified ?? self.dateModified
-        let datePaid = datePaid ?? nil
+        let datePaid = datePaid ?? self.datePaid
```

Comment:
> Will this break something in CPP?

I guess so because some tests fail now. 😓 

```swift

test_collect_payment_button_is_visible_and_primary_style_if_order_is_eligible_for_cash_on_delivery()

test_collect_payment_button_is_visible_if_order_is_eligible_for_cash_on_delivery_and_total_amount_is_greater_than_zero()
```

Reply:
Maybe we can put `nil` there and remove the protocol `GeneratedCopiable` from the `Order` model, as a first solution. What do you think?

Reply:
👋🏼  I completely missed the fact that it would be overwritten. 

The reason to update it was to be able to apply the lense to the datePaid property of a mock order. If the default value is nil, this would not set the datePaid to an actual nil value:

```
        let order = makeOrder().copy(datePaid: nil)
```

I think the long term solution would be to modify the way that `Models+Copiable.generated.swift` is generated so that the datePaid can actually be nullified, but in the meantime, it is okay to disable those two tests. 

If this seems like a reasonable approach, would you mind logging a ticket for this in the [payments board](https://github.com/woocommerce/woocommerce-ios/projects/30) please, so that we can get to it next week?

Reply:
Oh, I understand the use case now!

To nullify a field we wan use 
```swift
 let order = makeOrder().copy(datePaid: .some(nil))
```

The regular `.copy(datePaid: .some(nil))` is used to leave a value unmodified.

@fernandofgfer would you mind creating a separate PR fixing those tests, so we can merge this PR?

Reply:
Hi! I've reverted this change in this PR, in order to be merged. I will create a new PR to try to fix this issue!

Reply:
Hi again! You have the PR that fixes these tests here: https://github.com/woocommerce/woocommerce-ios/pull/4634!